rules/chan_policy_bounds_test.go

```diff
@@ -368,6 +368,64 @@ func TestChannelPolicyBoundsCheckers(t *testing.T) {
 			},
 			expectErr: true,
 		},
+		{
+			name: "channel opening sync in bounds",
+			uri:  "/lnrpc.Lightning/OpenChannelSync",
+			msg: &lnrpc.OpenChannelRequest{
+				UseBaseFee:  true,
+				BaseFee:     6,
+				UseFeeRate:  true,
+				FeeRate:     6000000,
+				MinHtlcMsat: 20,
+			},
+		},
+		{
+			name: "channel opening low high fee rate",
+			uri:  "/lnrpc.Lightning/OpenChannel",
+			msg: &lnrpc.OpenChannelRequest{
+				UseBaseFee:  true,
+				BaseFee:     6,
+				UseFeeRate:  true,
+				FeeRate:     11000000,
+				MinHtlcMsat: 20,
+			},
+			expectErr: true,
+		},
+		{
+			name: "channel opening low high base fee",
+			uri:  "/lnrpc.Lightning/OpenChannel",
+			msg: &lnrpc.OpenChannelRequest{
+				UseBaseFee:  true,
+				BaseFee:     11,
+				UseFeeRate:  true,
+				FeeRate:     6000000,
+				MinHtlcMsat: 20,
+			},
+			expectErr: true,
+		},
+		{
+			name: "batch channel opening low min htlc msat",
+			uri:  "/lnrpc.Lightning/BatchOpenChannel",
+			msg: &lnrpc.BatchOpenChannelRequest{
+				Channels: []*lnrpc.BatchOpenChannel{
+					{
+						UseBaseFee:  true,
+						BaseFee:     6,
+						UseFeeRate:  true,
+						FeeRate:     6000000,
+						MinHtlcMsat: 20,
+					},
+					{
+						UseBaseFee:  true,
+						BaseFee:     6,
+						UseFeeRate:  true,
+						FeeRate:     6000000,
+						MinHtlcMsat: 2,
+					},
+				},
+			},
+			expectErr: true,
```

Comment:
Same here, it would be nice with coverage of a `BatchOpenChannelRequest` that does succeed!